controllers/apps/components/utils.go

```diff
@@ -644,10 +644,15 @@ func updateKubeBlocksToolsImage(pc *corev1.Container) {
 
 func getImageName(image string) string {
 	subs := strings.Split(image, ":")
-	if len(subs) != 2 {
+	switch len(subs) {
+	case 2:
+		return subs[0]
+	case 3:
+		lastIndex := strings.LastIndex(image, ":")
+		return image[:lastIndex]
```

Comment:
seems not return the image name but image version? do test it.
write a ut for this func is better and encouraged.